api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -141,7 +145,7 @@ private DbUser saveUserWithConflictHandling(DbUser dbUser) {
     }
   }
 
-  private DbUser maybeInitializeUserWithTerra() {
+  private DbUser maybeInitializeUserWithTerraAndVwb() {
```

Comment:
suggest to have better seperation between Terra and VWB
`maybeInitializeUserWithTerra`
then
`maybeInitializeUserWithVwb`

Reply:
@yonghaoy  I made a couple of changes to address this, please take a look.